cypress/integration/purpose.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  givenIAmAt,
+  givenIHaveSelected,
+  iCanClickTheBackLinkToGoToPreviousPage,
+  requiredFieldErrorMessage,
+  thenIShouldSeeAnErrorMessageThatContains,
+  thenMyFocusMovesTo,
+  thenTheUrlShouldContain,
+  whenIClickContinue,
+  whenIClickOnTheErrorSummaryLinkContaining,
+} from "./common.spec";
+
+describe("As a beacon owner, I want to submit the purpose for my beacon", () => {
+  const thisPageUrl = "/register-a-beacon/purpose";
+  const previousPageUrl = "/register-a-beacon/envionment";
+
+  beforeEach(() => {
+    givenIAmAt(thisPageUrl);
+  });
+
+  xit("allows me to go back a page by following the 'back' button", () => {
+    iCanClickTheBackLinkToGoToPreviousPage(previousPageUrl);
+  });
+
+  xit("displays an error if no beacon use purpose is selected", () => {
+    whenIClickContinue();
+    thenIShouldSeeAnErrorMessageThatContains(requiredFieldErrorMessage);
+    whenIClickOnTheErrorSummaryLinkContaining(
+      "purpose",
+      requiredFieldErrorMessage
+    );
+    thenMyFocusMovesTo("#pleasure");
+  });
```

Comment:
@stuart-madetech I've just added an assertion that clicking on the error summary link shifts the user's focus to the relevant field, in common with other page tests.

There's an issue with our radio list component that the error summary link should shift focus to the first radio button, in this case `#pleasure`, but doesn't (issue #149).  Because of this issue, I've also skipped the test for now.